packages/ui-react/src/components/ErrorPage/ErrorPage.tsx

```diff
@@ -32,11 +32,12 @@ const ErrorPage = ({ title, message, learnMoreLabel, ...rest }: Props) => {
 
 export const ErrorPageWithoutTranslation = ({ title, children, message, learnMoreLabel, error, helpLink }: Props) => {
   const logo = useConfigStore((s) => s.config?.assets?.banner);
+  const alt = ''; // intentionally empty for a11y, because adjacent text alternative
```

Comment:
Because the logo doesn't add any value for screen readers, it's recommended to use an empty alt attribute: https://www.w3.org/WAI/tutorials/images/decorative/

We should use an alt attribute describing the image when dealing with other images (like a movie poster). But this is difficult because no good image description is available (descriptive !== "Sintel movie poster").